packages/edge/src/vercel-edge/index.ts

```diff
@@ -94,13 +119,14 @@ export function withEdgeMiddlewareAuth(
     });
 
     if (status === AuthStatus.Interstitial) {
-      return new NextResponse(interstitial, {
-        headers: { 'Content-Type': 'text/html', 'Auth-Result': errorReason || '' },
-        status: 401,
-      });
+      return interstitialResponse(interstitial as string, errorReason);
     }
 
     if (status === AuthStatus.SignedOut) {
+      if (options.strict) {
+        return signedOutResponse();
```

Comment:
I think we still want to pass errorReason here and set the Auth-Result error (we set it for interstitial and signed out states)

Reply:
Sure, good catch!